psautohint-2.4.0 broken on x86_64

### Describe the bug
A clear and concise description of what the bug is.

### Steps To Reproduce
Steps to reproduce the behavior:
Updated my `nixpkgs` (flake based on 600adcfdcc0c177cd69207ce1e05a9c37107da59).

### Expected behavior
Expected packages to build properly.

### Screenshots
```bash
Sourcing python-remove-tests-dir-hook
Sourcing python-catch-conflicts-hook.sh
Sourcing python-remove-bin-bytecode-hook.sh
Sourcing setuptools-build-hook
Using setuptoolsBuildPhase
Using setuptoolsShellHook
Sourcing pip-install-hook
Using pipInstallPhase
Sourcing python-imports-check-hook.sh
Using pythonImportsCheckPhase
Sourcing python-namespaces-hook
Sourcing python-catch-conflicts-hook.sh
Sourcing setuptools-check-hook
Using setuptoolsCheckPhase
Sourcing pytest-check-hook
Using pytestCheckPhase
Removing setuptoolsCheckPhase
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking sources
unpacking source archive /nix/store/hhfxfiizyixdm5rrwbcmq068lvmi3rf2-source
source root is source
setting SOURCE_DATE_EPOCH to timestamp 315619200 of file source/util/launch-asan.sh
@nix { "action": "setPhase", "phase": "patchPhase" }
patching sources
@nix { "action": "setPhase", "phase": "configurePhase" }
configuring
no configure script, doing nothing
@nix { "action": "setPhase", "phase": "buildPhase" }
building
Executing setuptoolsBuildPhase
/nix/store/yvv04lrgq2m4vkwlzs9jv720vj9jhdzy-python3.10-setuptools-65.6.3/lib/python3.10/site-packages/setuptools/config/setupcfg.py:508: SetuptoolsDeprecationWarning: The license_file parameter is deprecated, use license_files instead.
  warnings.warn(msg, warning_class)
running bdist_wheel
running build
running build_py
creating build
creating build/lib.linux-x86_64-cpython-310
creating build/lib.linux-x86_64-cpython-310/psautohint
copying python/psautohint/ufoFont.py -> build/lib.linux-x86_64-cpython-310/psautohint
copying python/psautohint/__init__.py -> build/lib.linux-x86_64-cpython-310/psautohint
copying python/psautohint/otfFont.py -> build/lib.linux-x86_64-cpython-310/psautohint
copying python/psautohint/fdTools.py -> build/lib.linux-x86_64-cpython-310/psautohint
copying python/psautohint/__main__.py -> build/lib.linux-x86_64-cpython-310/psautohint
copying python/psautohint/autohint.py -> build/lib.linux-x86_64-cpython-310/psautohint
running build_clib
building 'psautohint' library
creating build/temp.linux-x86_64-cpython-310
creating build/temp.linux-x86_64-cpython-310/libpsautohint
creating build/temp.linux-x86_64-cpython-310/libpsautohint/src
gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -I/nix/store/vshdx1ds1rmpl4by04i2g807zvkqyq8q-libxcrypt-4.4.33/include -fPIC -DAC_C_LIB_EXPORTS -Ilibpsautohint/include -c libpsautohint/src/ac.c -o build/temp.linux-x86_64-cpython-310/libpsautohint/src/ac.o
gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -I/nix/store/vshdx1ds1rmpl4by04i2g807zvkqyq8q-libxcrypt-4.4.33/include -fPIC -DAC_C_LIB_EXPORTS -Ilibpsautohint/include -c libpsautohint/src/acfixed.c -o build/temp.linux-x86_64-cpython-310/libpsautohint/src/acfixed.o
gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -I/nix/store/vshdx1ds1rmpl4by04i2g807zvkqyq8q-libxcrypt-4.4.33/include -fPIC -DAC_C_LIB_EXPORTS -Ilibpsautohint/include -c libpsautohint/src/auto.c -o build/temp.linux-x86_64-cpython-310/libpsautohint/src/auto.o
gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -I/nix/store/vshdx1ds1rmpl4by04i2g807zvkqyq8q-libxcrypt-4.4.33/include -fPIC -DAC_C_LIB_EXPORTS -Ilibpsautohint/include -c libpsautohint/src/bbox.c -o build/temp.linux-x86_64-cpython-310/libpsautohint/src/bbox.o
gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -I/nix/store/vshdx1ds1rmpl4by04i2g807zvkqyq8q-libxcrypt-4.4.33/include -fPIC -DAC_C_LIB_EXPORTS -Ilibpsautohint/include -c libpsautohint/src/buffer.c -o build/temp.linux-x86_64-cpython-310/libpsautohint/src/buffer.o
gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -I/nix/store/vshdx1ds1rmpl4by04i2g807zvkqyq8q-libxcrypt-4.4.33/include -fPIC -DAC_C_LIB_EXPORTS -Ilibpsautohint/include -c libpsautohint/src/charpath.c -o build/temp.linux-x86_64-cpython-310/libpsautohint/src/charpath.o
gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -I/nix/store/vshdx1ds1rmpl4by04i2g807zvkqyq8q-libxcrypt-4.4.33/include -fPIC -DAC_C_LIB_EXPORTS -Ilibpsautohint/include -c libpsautohint/src/charpathpriv.c -o build/temp.linux-x86_64-cpython-310/libpsautohint/src/charpathpriv.o
gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -I/nix/store/vshdx1ds1rmpl4by04i2g807zvkqyq8q-libxcrypt-4.4.33/include -fPIC -DAC_C_LIB_EXPORTS -Ilibpsautohint/include -c libpsautohint/src/charprop.c -o build/temp.linux-x86_64-cpython-310/libpsautohint/src/charprop.o
gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -I/nix/store/vshdx1ds1rmpl4by04i2g807zvkqyq8q-libxcrypt-4.4.33/include -fPIC -DAC_C_LIB_EXPORTS -Ilibpsautohint/include -c libpsautohint/src/check.c -o build/temp.linux-x86_64-cpython-310/libpsautohint/src/check.o
gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -I/nix/store/vshdx1ds1rmpl4by04i2g807zvkqyq8q-libxcrypt-4.4.33/include -fPIC -DAC_C_LIB_EXPORTS -Ilibpsautohint/include -c libpsautohint/src/control.c -o build/temp.linux-x86_64-cpython-310/libpsautohint/src/control.o
gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -I/nix/store/vshdx1ds1rmpl4by04i2g807zvkqyq8q-libxcrypt-4.4.33/include -fPIC -DAC_C_LIB_EXPORTS -Ilibpsautohint/include -c libpsautohint/src/eval.c -o build/temp.linux-x86_64-cpython-310/libpsautohint/src/eval.o
gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -I/nix/store/vshdx1ds1rmpl4by04i2g807zvkqyq8q-libxcrypt-4.4.33/include -fPIC -DAC_C_LIB_EXPORTS -Ilibpsautohint/include -c libpsautohint/src/fix.c -o build/temp.linux-x86_64-cpython-310/libpsautohint/src/fix.o
gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -I/nix/store/vshdx1ds1rmpl4by04i2g807zvkqyq8q-libxcrypt-4.4.33/include -fPIC -DAC_C_LIB_EXPORTS -Ilibpsautohint/include -c libpsautohint/src/flat.c -o build/temp.linux-x86_64-cpython-310/libpsautohint/src/flat.o
gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -I/nix/store/vshdx1ds1rmpl4by04i2g807zvkqyq8q-libxcrypt-4.4.33/include -fPIC -DAC_C_LIB_EXPORTS -Ilibpsautohint/include -c libpsautohint/src/fontinfo.c -o build/temp.linux-x86_64-cpython-310/libpsautohint/src/fontinfo.o
gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -I/nix/store/vshdx1ds1rmpl4by04i2g807zvkqyq8q-libxcrypt-4.4.33/include -fPIC -DAC_C_LIB_EXPORTS -Ilibpsautohint/include -c libpsautohint/src/gen.c -o build/temp.linux-x86_64-cpython-310/libpsautohint/src/gen.o
gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -I/nix/store/vshdx1ds1rmpl4by04i2g807zvkqyq8q-libxcrypt-4.4.33/include -fPIC -DAC_C_LIB_EXPORTS -Ilibpsautohint/include -c libpsautohint/src/head.c -o build/temp.linux-x86_64-cpython-310/libpsautohint/src/head.o
gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -I/nix/store/vshdx1ds1rmpl4by04i2g807zvkqyq8q-libxcrypt-4.4.33/include -fPIC -DAC_C_LIB_EXPORTS -Ilibpsautohint/include -c libpsautohint/src/logging.c -o build/temp.linux-x86_64-cpython-310/libpsautohint/src/logging.o
gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -I/nix/store/vshdx1ds1rmpl4by04i2g807zvkqyq8q-libxcrypt-4.4.33/include -fPIC -DAC_C_LIB_EXPORTS -Ilibpsautohint/include -c libpsautohint/src/memory.c -o build/temp.linux-x86_64-cpython-310/libpsautohint/src/memory.o
gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -I/nix/store/vshdx1ds1rmpl4by04i2g807zvkqyq8q-libxcrypt-4.4.33/include -fPIC -DAC_C_LIB_EXPORTS -Ilibpsautohint/include -c libpsautohint/src/merge.c -o build/temp.linux-x86_64-cpython-310/libpsautohint/src/merge.o
gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -I/nix/store/vshdx1ds1rmpl4by04i2g807zvkqyq8q-libxcrypt-4.4.33/include -fPIC -DAC_C_LIB_EXPORTS -Ilibpsautohint/include -c libpsautohint/src/misc.c -o build/temp.linux-x86_64-cpython-310/libpsautohint/src/misc.o
gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -I/nix/store/vshdx1ds1rmpl4by04i2g807zvkqyq8q-libxcrypt-4.4.33/include -fPIC -DAC_C_LIB_EXPORTS -Ilibpsautohint/include -c libpsautohint/src/optable.c -o build/temp.linux-x86_64-cpython-310/libpsautohint/src/optable.o
gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -I/nix/store/vshdx1ds1rmpl4by04i2g807zvkqyq8q-libxcrypt-4.4.33/include -fPIC -DAC_C_LIB_EXPORTS -Ilibpsautohint/include -c libpsautohint/src/pick.c -o build/temp.linux-x86_64-cpython-310/libpsautohint/src/pick.o
gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -I/nix/store/vshdx1ds1rmpl4by04i2g807zvkqyq8q-libxcrypt-4.4.33/include -fPIC -DAC_C_LIB_EXPORTS -Ilibpsautohint/include -c libpsautohint/src/psautohint.c -o build/temp.linux-x86_64-cpython-310/libpsautohint/src/psautohint.o
gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -I/nix/store/vshdx1ds1rmpl4by04i2g807zvkqyq8q-libxcrypt-4.4.33/include -fPIC -DAC_C_LIB_EXPORTS -Ilibpsautohint/include -c libpsautohint/src/read.c -o build/temp.linux-x86_64-cpython-310/libpsautohint/src/read.o
gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -I/nix/store/vshdx1ds1rmpl4by04i2g807zvkqyq8q-libxcrypt-4.4.33/include -fPIC -DAC_C_LIB_EXPORTS -Ilibpsautohint/include -c libpsautohint/src/report.c -o build/temp.linux-x86_64-cpython-310/libpsautohint/src/report.o
gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -I/nix/store/vshdx1ds1rmpl4by04i2g807zvkqyq8q-libxcrypt-4.4.33/include -fPIC -DAC_C_LIB_EXPORTS -Ilibpsautohint/include -c libpsautohint/src/shuffle.c -o build/temp.linux-x86_64-cpython-310/libpsautohint/src/shuffle.o
gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -I/nix/store/vshdx1ds1rmpl4by04i2g807zvkqyq8q-libxcrypt-4.4.33/include -fPIC -DAC_C_LIB_EXPORTS -Ilibpsautohint/include -c libpsautohint/src/stemreport.c -o build/temp.linux-x86_64-cpython-310/libpsautohint/src/stemreport.o
gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -I/nix/store/vshdx1ds1rmpl4by04i2g807zvkqyq8q-libxcrypt-4.4.33/include -fPIC -DAC_C_LIB_EXPORTS -Ilibpsautohint/include -c libpsautohint/src/write.c -o build/temp.linux-x86_64-cpython-310/libpsautohint/src/write.o
ar rcs build/temp.linux-x86_64-cpython-310/libpsautohint.a build/temp.linux-x86_64-cpython-310/libpsautohint/src/ac.o build/temp.linux-x86_64-cpython-310/libpsautohint/src/acfixed.o build/temp.linux-x86_64-cpython-310/libpsautohint/src/auto.o build/temp.linux-x86_64-cpython-310/libpsautohint/src/bbox.o build/temp.linux-x86_64-cpython-310/libpsautohint/src/buffer.o build/temp.linux-x86_64-cpython-310/libpsautohint/src/charpath.o build/temp.linux-x86_64-cpython-310/libpsautohint/src/charpathpriv.o build/temp.linux-x86_64-cpython-310/libpsautohint/src/charprop.o build/temp.linux-x86_64-cpython-310/libpsautohint/src/check.o build/temp.linux-x86_64-cpython-310/libpsautohint/src/control.o build/temp.linux-x86_64-cpython-310/libpsautohint/src/eval.o build/temp.linux-x86_64-cpython-310/libpsautohint/src/fix.o build/temp.linux-x86_64-cpython-310/libpsautohint/src/flat.o build/temp.linux-x86_64-cpython-310/libpsautohint/src/fontinfo.o build/temp.linux-x86_64-cpython-310/libpsautohint/src/gen.o build/temp.linux-x86_64-cpython-310/libpsautohint/src/head.o build/temp.linux-x86_64-cpython-310/libpsautohint/src/logging.o build/temp.linux-x86_64-cpython-310/libpsautohint/src/memory.o build/temp.linux-x86_64-cpython-310/libpsautohint/src/merge.o build/temp.linux-x86_64-cpython-310/libpsautohint/src/misc.o build/temp.linux-x86_64-cpython-310/libpsautohint/src/optable.o build/temp.linux-x86_64-cpython-310/libpsautohint/src/pick.o build/temp.linux-x86_64-cpython-310/libpsautohint/src/psautohint.o build/temp.linux-x86_64-cpython-310/libpsautohint/src/read.o build/temp.linux-x86_64-cpython-310/libpsautohint/src/report.o build/temp.linux-x86_64-cpython-310/libpsautohint/src/shuffle.o build/temp.linux-x86_64-cpython-310/libpsautohint/src/stemreport.o build/temp.linux-x86_64-cpython-310/libpsautohint/src/write.o
running build_ext
building 'psautohint._psautohint' extension
creating build/temp.linux-x86_64-cpython-310/python
creating build/temp.linux-x86_64-cpython-310/python/psautohint
gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -I/nix/store/vshdx1ds1rmpl4by04i2g807zvkqyq8q-libxcrypt-4.4.33/include -fPIC -Ilibpsautohint/include -I/nix/store/sp5x6s8n36gjlwck74xhj1i61p66vcpa-python3-3.10.9/include/python3.10 -c python/psautohint/_psautohint.c -o build/temp.linux-x86_64-cpython-310/python/psautohint/_psautohint.o
gcc -shared -L/nix/store/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee-zlib-1.2.13/lib -L/nix/store/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee-bzip2-1.0.8/lib -L/nix/store/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee-expat-2.5.0/lib -L/nix/store/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee-xz-5.4.1/lib -L/nix/store/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee-libffi-3.4.4/lib -L/nix/store/vshdx1ds1rmpl4by04i2g807zvkqyq8q-libxcrypt-4.4.33/lib -L/nix/store/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee-gdbm-1.23/lib -L/nix/store/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee-sqlite-3.40.1/lib -L/nix/store/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee-readline-8.2p1/lib -L/nix/store/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee-ncurses-6.4/lib -L/nix/store/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee-openssl-3.0.7/lib -L/nix/store/mj0wl2j1jdqj1vx1ljdlar8x9kp1bvyc-tzdata-2022g/lib -L/nix/store/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee-zlib-1.2.13/lib -L/nix/store/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee-bzip2-1.0.8/lib -L/nix/store/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee-expat-2.5.0/lib -L/nix/store/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee-xz-5.4.1/lib -L/nix/store/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee-libffi-3.4.4/lib -L/nix/store/vshdx1ds1rmpl4by04i2g807zvkqyq8q-libxcrypt-4.4.33/lib -L/nix/store/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee-gdbm-1.23/lib -L/nix/store/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee-sqlite-3.40.1/lib -L/nix/store/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee-readline-8.2p1/lib -L/nix/store/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee-ncurses-6.4/lib -L/nix/store/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee-openssl-3.0.7/lib -L/nix/store/mj0wl2j1jdqj1vx1ljdlar8x9kp1bvyc-tzdata-2022g/lib build/temp.linux-x86_64-cpython-310/python/psautohint/_psautohint.o -L/nix/store/sp5x6s8n36gjlwck74xhj1i61p66vcpa-python3-3.10.9/lib -Lbuild/temp.linux-x86_64-cpython-310 -lpsautohint -o build/lib.linux-x86_64-cpython-310/psautohint/_psautohint.cpython-310-x86_64-linux-gnu.so
/nix/store/yvv04lrgq2m4vkwlzs9jv720vj9jhdzy-python3.10-setuptools-65.6.3/lib/python3.10/site-packages/setuptools/command/install.py:34: SetuptoolsDeprecationWarning: setup.py install is deprecated. Use build and pip and other standards-based tools.
  warnings.warn(
installing to build/bdist.linux-x86_64/wheel
running install
running install_lib
creating build/bdist.linux-x86_64
creating build/bdist.linux-x86_64/wheel
creating build/bdist.linux-x86_64/wheel/psautohint
copying build/lib.linux-x86_64-cpython-310/psautohint/_psautohint.cpython-310-x86_64-linux-gnu.so -> build/bdist.linux-x86_64/wheel/psautohint
copying build/lib.linux-x86_64-cpython-310/psautohint/ufoFont.py -> build/bdist.linux-x86_64/wheel/psautohint
copying build/lib.linux-x86_64-cpython-310/psautohint/__init__.py -> build/bdist.linux-x86_64/wheel/psautohint
copying build/lib.linux-x86_64-cpython-310/psautohint/otfFont.py -> build/bdist.linux-x86_64/wheel/psautohint
copying build/lib.linux-x86_64-cpython-310/psautohint/fdTools.py -> build/bdist.linux-x86_64/wheel/psautohint
copying build/lib.linux-x86_64-cpython-310/psautohint/__main__.py -> build/bdist.linux-x86_64/wheel/psautohint
copying build/lib.linux-x86_64-cpython-310/psautohint/autohint.py -> build/bdist.linux-x86_64/wheel/psautohint
running install_egg_info
running egg_info
creating python/psautohint.egg-info
writing python/psautohint.egg-info/PKG-INFO
writing dependency_links to python/psautohint.egg-info/dependency_links.txt
writing entry points to python/psautohint.egg-info/entry_points.txt
writing requirements to python/psautohint.egg-info/requires.txt
writing top-level names to python/psautohint.egg-info/top_level.txt
writing manifest file 'python/psautohint.egg-info/SOURCES.txt'
reading manifest file 'python/psautohint.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
adding license file 'LICENSE'
writing manifest file 'python/psautohint.egg-info/SOURCES.txt'
Copying python/psautohint.egg-info to build/bdist.linux-x86_64/wheel/psautohint-2.4.0-py3.10.egg-info
running install_scripts
creating build/bdist.linux-x86_64/wheel/psautohint-2.4.0.dist-info/WHEEL
creating 'dist/psautohint-2.4.0-cp310-cp310-linux_x86_64.whl' and adding 'build/bdist.linux-x86_64/wheel' to it
adding 'psautohint/__init__.py'
adding 'psautohint/__main__.py'
adding 'psautohint/_psautohint.cpython-310-x86_64-linux-gnu.so'
adding 'psautohint/autohint.py'
adding 'psautohint/fdTools.py'
adding 'psautohint/otfFont.py'
adding 'psautohint/ufoFont.py'
adding 'psautohint-2.4.0.dist-info/LICENSE'
adding 'psautohint-2.4.0.dist-info/METADATA'
adding 'psautohint-2.4.0.dist-info/WHEEL'
adding 'psautohint-2.4.0.dist-info/entry_points.txt'
adding 'psautohint-2.4.0.dist-info/top_level.txt'
adding 'psautohint-2.4.0.dist-info/RECORD'
removing build/bdist.linux-x86_64/wheel
Finished executing setuptoolsBuildPhase
@nix { "action": "setPhase", "phase": "installPhase" }
installing
Executing pipInstallPhase
/build/source/dist /build/source
Processing ./psautohint-2.4.0-cp310-cp310-linux_x86_64.whl
Requirement already satisfied: fonttools[ufo]>=4.22.0 in /nix/store/fhdf0q9ql3qaa9jws8kvc9syv3xcfs11-python3.10-fonttools-4.38.0/lib/python3.10/site-packages (from psautohint==2.4.0) (4.38.0)
Requirement already satisfied: fs<3,>=2.2.0 in /nix/store/ql7lh3kzl90dp2mwbgwk1fq5hi45czg7-python3.10-fs-2.4.16/lib/python3.10/site-packages (from fonttools[ufo]>=4.22.0->psautohint==2.4.0) (2.4.16)
Requirement already satisfied: appdirs~=1.4.3 in /nix/store/9kg14rflqayhjf81qla9mw7y2qyb8imc-python3.10-appdirs-1.4.4/lib/python3.10/site-packages (from fs<3,>=2.2.0->fonttools[ufo]>=4.22.0->psautohint==2.4.0) (1.4.4)
Requirement already satisfied: setuptools in /nix/store/yvv04lrgq2m4vkwlzs9jv720vj9jhdzy-python3.10-setuptools-65.6.3/lib/python3.10/site-packages (from fs<3,>=2.2.0->fonttools[ufo]>=4.22.0->psautohint==2.4.0) (65.6.3.post0)
Requirement already satisfied: six~=1.10 in /nix/store/f7674syc19gcak5ja2m7kh0hnmm2cmp9-python3.10-six-1.16.0/lib/python3.10/site-packages (from fs<3,>=2.2.0->fonttools[ufo]>=4.22.0->psautohint==2.4.0) (1.16.0)
Installing collected packages: psautohint
Successfully installed psautohint-2.4.0
/build/source
Finished executing pipInstallPhase
@nix { "action": "setPhase", "phase": "pythonOutputDistPhase" }
pythonOutputDistPhase
Executing pythonOutputDistPhase
Finished executing pythonOutputDistPhase
@nix { "action": "setPhase", "phase": "fixupPhase" }
post-installation fixup
shrinking RPATHs of ELF executables and libraries in /nix/store/xc83qqg0yqbqzr8hwlh9nqkg8p7cbhix-python3.10-psautohint-2.4.0
shrinking /nix/store/xc83qqg0yqbqzr8hwlh9nqkg8p7cbhix-python3.10-psautohint-2.4.0/lib/python3.10/site-packages/psautohint/_psautohint.cpython-310-x86_64-linux-gnu.so
checking for references to /build/ in /nix/store/xc83qqg0yqbqzr8hwlh9nqkg8p7cbhix-python3.10-psautohint-2.4.0...
patching script interpreter paths in /nix/store/xc83qqg0yqbqzr8hwlh9nqkg8p7cbhix-python3.10-psautohint-2.4.0
stripping (with command strip and flags -S) in  /nix/store/xc83qqg0yqbqzr8hwlh9nqkg8p7cbhix-python3.10-psautohint-2.4.0/lib /nix/store/xc83qqg0yqbqzr8hwlh9nqkg8p7cbhix-python3.10-psautohint-2.4.0/bin
shrinking RPATHs of ELF executables and libraries in /nix/store/s3g9vp3q22nc109gkjzsb9b57zs4i4yl-python3.10-psautohint-2.4.0-dist
checking for references to /build/ in /nix/store/s3g9vp3q22nc109gkjzsb9b57zs4i4yl-python3.10-psautohint-2.4.0-dist...
patching script interpreter paths in /nix/store/s3g9vp3q22nc109gkjzsb9b57zs4i4yl-python3.10-psautohint-2.4.0-dist
Rewriting #!/nix/store/sp5x6s8n36gjlwck74xhj1i61p66vcpa-python3-3.10.9/bin/python3.10 to #!/nix/store/sp5x6s8n36gjlwck74xhj1i61p66vcpa-python3-3.10.9
wrapping `/nix/store/xc83qqg0yqbqzr8hwlh9nqkg8p7cbhix-python3.10-psautohint-2.4.0/bin/psstemhist'...
Rewriting #!/nix/store/sp5x6s8n36gjlwck74xhj1i61p66vcpa-python3-3.10.9/bin/python3.10 to #!/nix/store/sp5x6s8n36gjlwck74xhj1i61p66vcpa-python3-3.10.9
wrapping `/nix/store/xc83qqg0yqbqzr8hwlh9nqkg8p7cbhix-python3.10-psautohint-2.4.0/bin/psautohint'...
Executing pythonRemoveTestsDir
Finished executing pythonRemoveTestsDir
@nix { "action": "setPhase", "phase": "installCheckPhase" }
running install tests
no Makefile or custom installCheckPhase, doing nothing
@nix { "action": "setPhase", "phase": "pythonCatchConflictsPhase" }
pythonCatchConflictsPhase
@nix { "action": "setPhase", "phase": "pythonRemoveBinBytecodePhase" }
pythonRemoveBinBytecodePhase
@nix { "action": "setPhase", "phase": "pythonImportsCheckPhase" }
pythonImportsCheckPhase
Executing pythonImportsCheckPhase
@nix { "action": "setPhase", "phase": "pytestXdistHook" }
pytestXdistHook
@nix { "action": "setPhase", "phase": "pytestCheckPhase" }
pytestCheckPhase
Executing pytestCheckPhase
[1m============================= test session starts ==============================[0m
platform linux -- Python 3.10.9, pytest-7.2.0, pluggy-1.0.0 -- /nix/store/sp5x6s8n36gjlwck74xhj1i61p66vcpa-python3-3.10.9/bin/python3.10
cachedir: .pytest_cache
rootdir: /build/source, configfile: pytest.ini, testpaths: tests
plugins: cov-4.0.0, xdist-3.1.0
[1m
gw0 I / gw1 I / gw2 I / gw3 I / gw4 I / gw5 I / gw6 I / gw7 I[0m[1m
[gw0] linux Python 3.10.9 cwd: /build/source                 
[0m[1m
gw0 C / gw1 I / gw2 I / gw3 I / gw4 I / gw5 I / gw6 I / gw7 I[0m[1m
[gw1] linux Python 3.10.9 cwd: /build/source                 
[0m[1m
gw0 C / gw1 C / gw2 I / gw3 I / gw4 I / gw5 I / gw6 I / gw7 I[0m[1m
[gw2] linux Python 3.10.9 cwd: /build/source                 
[0m[1m
gw0 C / gw1 C / gw2 C / gw3 I / gw4 I / gw5 I / gw6 I / gw7 I[0m[1m
[gw3] linux Python 3.10.9 cwd: /build/source                 
[0m[1m
gw0 C / gw1 C / gw2 C / gw3 C / gw4 I / gw5 I / gw6 I / gw7 I[0m[1m
[gw4] linux Python 3.10.9 cwd: /build/source                 
[0m[1m
gw0 C / gw1 C / gw2 C / gw3 C / gw4 C / gw5 I / gw6 I / gw7 I[0m[1m
[gw5] linux Python 3.10.9 cwd: /build/source                 
[0m[1m
gw0 C / gw1 C / gw2 C / gw3 C / gw4 C / gw5 C / gw6 I / gw7 I[0m[1m
[gw6] linux Python 3.10.9 cwd: /build/source                 
[0m[1m
gw0 C / gw1 C / gw2 C / gw3 C / gw4 C / gw5 C / gw6 C / gw7 I[0m[1m
[gw7] linux Python 3.10.9 cwd: /build/source                 
[0m[1m
gw0 C / gw1 C / gw2 C / gw3 C / gw4 C / gw5 C / gw6 C / gw7 C[0m[1m
[gw0] Python 3.10.9 (main, Dec  6 2022, 18:44:57) [GCC 11.3.0]
[0m[1m
gw0 ok / gw1 C / gw2 C / gw3 C / gw4 C / gw5 C / gw6 C / gw7 C[0m[1m
[gw1] Python 3.10.9 (main, Dec  6 2022, 18:44:57) [GCC 11.3.0]
[0m[1m
gw0 ok / gw1 ok / gw2 C / gw3 C / gw4 C / gw5 C / gw6 C / gw7 C[0m[1m
[gw2] Python 3.10.9 (main, Dec  6 2022, 18:44:57) [GCC 11.3.0] 
[0m[1m
gw0 ok / gw1 ok / gw2 ok / gw3 C / gw4 C / gw5 C / gw6 C / gw7 C[0m[1m
[gw3] Python 3.10.9 (main, Dec  6 2022, 18:44:57) [GCC 11.3.0]  
[0m[1m
gw0 ok / gw1 ok / gw2 ok / gw3 ok / gw4 C / gw5 C / gw6 C / gw7 C[0m[1m
gw0 [180] / gw1 ok / gw2 ok / gw3 ok / gw4 C / gw5 C / gw6 C / gw7 C[0m[1m
[gw4] Python 3.10.9 (main, Dec  6 2022, 18:44:57) [GCC 11.3.0]      
[0m[1m
gw0 [180] / gw1 ok / gw2 ok / gw3 ok / gw4 ok / gw5 C / gw6 C / gw7 C[0m[1m
gw0 [180] / gw1 [180] / gw2 ok / gw3 ok / gw4 ok / gw5 C / gw6 C / gw7 C[0m[1m
[gw5] Python 3.10.9 (main, Dec  6 2022, 18:44:57) [GCC 11.3.0]          
[0m[1m
gw0 [180] / gw1 [180] / gw2 ok / gw3 ok / gw4 ok / gw5 ok / gw6 C / gw7 C[0m[1m
gw0 [180] / gw1 [180] / gw2 [180] / gw3 ok / gw4 ok / gw5 ok / gw6 C / gw7 C[0m[1m
gw0 [180] / gw1 [180] / gw2 [180] / gw3 ok / gw4 [180] / gw5 ok / gw6 C / gw7 C[0m[1m
[gw6] Python 3.10.9 (main, Dec  6 2022, 18:44:57) [GCC 11.3.0]                 
[0m[1m
gw0 [180] / gw1 [180] / gw2 [180] / gw3 ok / gw4 [180] / gw5 ok / gw6 ok / gw7 C[0m[1m
gw0 [180] / gw1 [180] / gw2 [180] / gw3 [180] / gw4 [180] / gw5 ok / gw6 ok / gw7 C[0m[1m
[gw7] Python 3.10.9 (main, Dec  6 2022, 18:44:57) [GCC 11.3.0]                     
[0m[1m
gw0 [180] / gw1 [180] / gw2 [180] / gw3 [180] / gw4 [180] / gw5 ok / gw6 ok / gw7 ok[0m[1m
gw0 [180] / gw1 [180] / gw2 [180] / gw3 [180] / gw4 [180] / gw5 [180] / gw6 ok / gw7 ok[0m[1m
gw0 [180] / gw1 [180] / gw2 [180] / gw3 [180] / gw4 [180] / gw5 [180] / gw6 [180] / gw7 ok[0m[1m
gw0 [180] / gw1 [180] / gw2 [180] / gw3 [180] / gw4 [180] / gw5 [180] / gw6 [180] / gw7 [180][0m
scheduling tests via LoadScheduling

tests/integration/test_cli.py::test_type1_raises[tests/integration/data/dummy/font.ps] 
tests/integration/test_cli.py::test_basic[tests/integration/data/dummy/font.cff] 
tests/integration/test_cli.py::test_missing_glyph_list[FOO-BAR] 
tests/integration/test_cli.py::test_outpath[tests/integration/data/dummy/font.ufo] 
tests/integration/test_cli.py::test_glyph_list[a,b,c] 
tests/integration/test_cli.py::test_missing_cff_table1 
tests/integration/test_cli.py::test_cid_glyph_list[cid0,cid1,cid2] 
[gw6][36m [  0%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_missing_cff_table1 
tests/integration/test_cli.py::test_missing_cff_table2 
[gw5][36m [  1%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_missing_glyph_list[FOO-BAR] 
tests/integration/test_cli.py::test_option[font.otf---exclude-glyphs-file-glyphs.txt] 
tests/integration/test_cli.py::test_missing_glyph_list[FOO-a] 
[gw6][36m [  1%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_missing_cff_table2 
[gw2][36m [  2%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_type1_raises[tests/integration/data/dummy/font.ps] 
tests/integration/test_cli.py::test_type1_raises[tests/integration/data/dummy/font.pfa] 
tests/integration/test_cli.py::test_option[font.ufo---exclude-glyphs-file-glyphs.txt] 
[gw2][36m [  2%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_type1_raises[tests/integration/data/dummy/font.pfa] 
tests/integration/test_cli.py::test_type1_supported[tests/integration/data/dummy/font.pfb] 
[gw2][36m [  3%] [0m[33mSKIPPED[0m tests/integration/test_cli.py::test_type1_supported[tests/integration/data/dummy/font.pfb] 
tests/integration/test_cli.py::test_type1_supported[tests/integration/data/dummy/font.ps] 
[gw2][36m [  3%] [0m[33mSKIPPED[0m tests/integration/test_cli.py::test_type1_supported[tests/integration/data/dummy/font.ps] 
[gw5][36m [  4%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_missing_glyph_list[FOO-a] 
tests/integration/test_cli.py::test_type1_supported[tests/integration/data/dummy/font.pfa] 
[gw2][36m [  5%] [0m[33mSKIPPED[0m tests/integration/test_cli.py::test_type1_supported[tests/integration/data/dummy/font.pfa] 
tests/integration/test_cli.py::test_argumentless_option[font.ufo---verbose] 
tests/integration/test_cli.py::test_missing_glyph_list[a-BAR] 
[gw5][36m [  5%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_missing_glyph_list[a-BAR] 
tests/integration/test_cli.py::test_unsupported_format[tests/integration/data/dummy/fontinfo] 
[gw5][36m [  6%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_unsupported_format[tests/integration/data/dummy/fontinfo] 
tests/integration/test_cli.py::test_unsupported_format[tests/integration/data] 
[gw5][36m [  6%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_unsupported_format[tests/integration/data] 
tests/integration/test_cli.py::test_argumentless_option[font.otf---no-flex] 
[gw3][36m [  7%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_glyph_list[a,b,c] 
tests/integration/test_cli.py::test_glyph_list[a-z] 
[gw0][36m [  7%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_basic[tests/integration/data/dummy/font.cff] 
[gw5][36m [  8%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_argumentless_option[font.otf---no-flex] 
tests/integration/test_cli.py::test_argumentless_option[font.otf---no-hint-sub] 
tests/integration/test_cli.py::test_basic[tests/integration/data/dummy/font.otf] 
[gw7][36m [  8%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_option[font.otf---exclude-glyphs-file-glyphs.txt] 
tests/integration/test_cli.py::test_option[font.otf---fontinfo-file-fontinfo] 
[gw3][36m [  9%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_glyph_list[a-z] 
tests/integration/test_cli.py::test_glyph_list[FOO,BAR,a] 
[gw3][36m [ 10%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_glyph_list[FOO,BAR,a] 
tests/integration/test_cli.py::test_cid_glyph_list[/0,/1,/2] 
[gw5][36m [ 10%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_argumentless_option[font.otf---no-hint-sub] 
[gw0][36m [ 11%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_basic[tests/integration/data/dummy/font.otf] 
tests/integration/test_cli.py::test_basic[tests/integration/data/dummy/font.ufo] 
tests/integration/test_cli.py::test_argumentless_option[font.otf---no-zones-stems] 
[gw1][36m [ 11%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_outpath[tests/integration/data/dummy/font.ufo] 
tests/integration/test_cli.py::test_multi_different_formats 
[gw6][36m [ 12%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_option[font.ufo---exclude-glyphs-file-glyphs.txt] 
tests/integration/test_cli.py::test_option[font.ufo---fontinfo-file-fontinfo] 
[gw1][36m [ 12%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_multi_different_formats 
tests/integration/test_cli.py::test_multi_reference_is_input 
[gw7][36m [ 13%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_option[font.otf---fontinfo-file-fontinfo] 
tests/integration/test_cli.py::test_option[font.otf---glyphs-file-glyphs.txt] 
[gw2][36m [ 13%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_argumentless_option[font.ufo---verbose] 
tests/integration/test_cli.py::test_argumentless_option[font.ufo---write-to-default-layer] 
[gw7][36m [ 14%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_option[font.otf---glyphs-file-glyphs.txt] 
tests/integration/test_cli.py::test_argumentless_option[font.ufo---allow-changes] 
[gw1][36m [ 15%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_multi_reference_is_input 
tests/integration/test_cli.py::test_multi_reference_is_duplicated 
[gw5][36m [ 15%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_argumentless_option[font.otf---no-zones-stems] 
tests/integration/test_cli.py::test_argumentless_option[font.otf---print-list-fddict] 
[gw5][36m [ 16%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_argumentless_option[font.otf---print-list-fddict] 
tests/integration/test_cli.py::test_argumentless_option[font.otf---report-only] 
[gw1][36m [ 16%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_multi_reference_is_duplicated 
tests/integration/test_cli.py::test_type1_raises[tests/integration/data/dummy/font.pfb] 
[gw1][36m [ 17%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_type1_raises[tests/integration/data/dummy/font.pfb] 
tests/integration/test_cli.py::test_doc_option[--info] 
[gw1][36m [ 17%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_doc_option[--info] 
tests/integration/test_cli.py::test_doc_option[--version] 
[gw1][36m [ 18%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_doc_option[--version] 
tests/integration/test_cli.py::test_no_fddict 
[gw5][36m [ 18%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_argumentless_option[font.otf---report-only] 
tests/integration/test_cli.py::test_argumentless_option[font.otf---verbose] 
[gw5][36m [ 19%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_argumentless_option[font.otf---verbose] 
tests/integration/test_cli.py::test_argumentless_option[font.otf---write-to-default-layer] 
[gw0][36m [ 20%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_basic[tests/integration/data/dummy/font.ufo] 
tests/integration/test_cli.py::test_outpath[tests/integration/data/dummy/font.cff] 
[gw4][36m [ 20%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_cid_glyph_list[cid0,cid1,cid2] 
tests/integration/test_cli.py::test_cid_glyph_list[cid0-cid10] 
[gw6][36m [ 21%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_option[font.ufo---fontinfo-file-fontinfo] 
tests/integration/test_cli.py::test_option[font.ufo---glyphs-file-glyphs.txt] 
[gw5][36m [ 21%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_argumentless_option[font.otf---write-to-default-layer] 
tests/integration/test_cli.py::test_argumentless_option[font.otf--vv] 
[gw6][36m [ 22%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_option[font.ufo---glyphs-file-glyphs.txt] 
tests/integration/test_cli.py::test_argumentless_option[font.ufo---no-flex] 
[gw2][36m [ 22%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_argumentless_option[font.ufo---write-to-default-layer] 
tests/integration/test_cli.py::test_argumentless_option[font.ufo--vv] 
[gw7][36m [ 23%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_argumentless_option[font.ufo---allow-changes] 
tests/integration/test_cli.py::test_argumentless_option[font.ufo---decimal] 
[gw5][36m [ 23%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_argumentless_option[font.otf--vv] 
tests/integration/test_cli.py::test_print_fddict 
[gw5][36m [ 24%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_print_fddict 
tests/integration/test_cli.py::test_doc_option[--doc-fddict] 
[gw5][36m [ 25%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_doc_option[--doc-fddict] 
tests/integration/test_cli.py::test_doc_option[--help] 
[gw5][36m [ 25%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_doc_option[--help] 
tests/integration/test_cli.py::test_outpath_order[tests/integration/data/dummy/font.cff] 
[gw0][36m [ 26%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_outpath[tests/integration/data/dummy/font.cff] 
tests/integration/test_cli.py::test_outpath[tests/integration/data/dummy/font.otf] 
[gw3][36m [ 26%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_cid_glyph_list[/0,/1,/2] 
tests/integration/test_cli.py::test_cid_glyph_list[/0-/10] 
[gw5][36m [ 27%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_outpath_order[tests/integration/data/dummy/font.cff] 
tests/integration/test_cli.py::test_outpath_order[tests/integration/data/dummy/font.otf] 
[gw7][36m [ 27%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_argumentless_option[font.ufo---decimal] 
tests/integration/test_cli.py::test_invalid_input_path 
[gw7][36m [ 28%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_invalid_input_path 
tests/integration/test_cli.py::test_invalid_save_path 
[gw7][36m [ 28%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_invalid_save_path 
[gw0][36m [ 29%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_outpath[tests/integration/data/dummy/font.otf] 
[gw6][36m [ 30%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_argumentless_option[font.ufo---no-flex] 
tests/integration/test_cli.py::test_stemhist[report_zones] 
tests/integration/test_cli.py::test_multi_order_unequal 
tests/integration/test_cli.py::test_argumentless_option[font.ufo---no-hint-sub] 
[gw0][36m [ 30%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_multi_order_unequal 
tests/integration/test_cli.py::test_lack_of_input_raises 
[gw0][36m [ 31%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_lack_of_input_raises 
tests/integration/test_hint.py::test_cff[tests/integration/data/dummy/font.cff] 
[gw5][36m [ 31%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_outpath_order[tests/integration/data/dummy/font.otf] 
tests/integration/test_cli.py::test_outpath_order[tests/integration/data/dummy/font.ufo] 
[gw2][36m [ 32%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_argumentless_option[font.ufo--vv] 
tests/integration/test_cli.py::test_argumentless_option[font.otf---allow-changes] 
[gw7][36m [ 32%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_stemhist[report_zones] 
tests/integration/test_cli.py::test_stemhist[report_stems] 
[gw7][36m [ 33%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_stemhist[report_stems] 
tests/integration/test_cli.py::test_stemhist[report_stems,all_stems] 
[gw0][36m [ 33%] [0m[32mPASSED[0m tests/integration/test_hint.py::test_cff[tests/integration/data/dummy/font.cff] 
tests/integration/test_hint.py::test_type1_raises[tests/integration/data/dummy/font.pfb] 
[gw7][36m [ 34%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_stemhist[report_stems,all_stems] 
[gw0][36m [ 35%] [0m[32mPASSED[0m tests/integration/test_hint.py::test_type1_raises[tests/integration/data/dummy/font.pfb] 
tests/integration/test_cli.py::test_stemhist[report_stems,glyphs] 
tests/integration/test_hint.py::test_type1_raises[tests/integration/data/dummy/font.ps] 
[gw0][36m [ 35%] [0m[32mPASSED[0m tests/integration/test_hint.py::test_type1_raises[tests/integration/data/dummy/font.ps] 
tests/integration/test_hint.py::test_counter_glyphs 
[gw2][36m [ 36%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_argumentless_option[font.otf---allow-changes] 
tests/integration/test_cli.py::test_argumentless_option[font.otf---decimal] 
[gw7][36m [ 36%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_stemhist[report_stems,glyphs] 
tests/integration/test_hint.py::test_missing_cff_table 
[gw7][36m [ 37%] [0m[32mPASSED[0m tests/integration/test_hint.py::test_missing_cff_table 
tests/integration/test_hint.py::test_type1_bad[tests/integration/data/dummy/bad_font.ps] 
[gw7][36m [ 37%] [0m[32mPASSED[0m tests/integration/test_hint.py::test_type1_bad[tests/integration/data/dummy/bad_font.ps] 
tests/integration/test_hint.py::test_type1_bad[tests/integration/data/dummy/bad_font.pfa] 
[gw7][36m [ 38%] [0m[32mPASSED[0m tests/integration/test_hint.py::test_type1_bad[tests/integration/data/dummy/bad_font.pfa] 
tests/integration/test_hint.py::test_type1_bad[tests/integration/data/dummy/bad_font.pfb] 
[gw7][36m [ 38%] [0m[32mPASSED[0m tests/integration/test_hint.py::test_type1_bad[tests/integration/data/dummy/bad_font.pfb] 
tests/integration/test_hint.py::test_bad_privatedict[tests/integration/data/dummy/bad_privatedict_bluevalues.ufo] 
[gw7][36m [ 39%] [0m[32mPASSED[0m tests/integration/test_hint.py::test_bad_privatedict[tests/integration/data/dummy/bad_privatedict_bluevalues.ufo] 
tests/integration/test_hint.py::test_bad_privatedict_accept[tests/integration/data/dummy/bad_privatedict_bluevalues.otf] 
[gw7][36m [ 40%] [0m[32mPASSED[0m tests/integration/test_hint.py::test_bad_privatedict_accept[tests/integration/data/dummy/bad_privatedict_bluevalues.otf] 
tests/integration/test_hint.py::test_bad_privatedict_accept[tests/integration/data/dummy/bad_privatedict_vstems.ufo] 
[gw6][36m [ 40%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_argumentless_option[font.ufo---no-hint-sub] 
tests/integration/test_cli.py::test_argumentless_option[font.ufo---no-zones-stems] 
[gw7][36m [ 41%] [0m[32mPASSED[0m tests/integration/test_hint.py::test_bad_privatedict_accept[tests/integration/data/dummy/bad_privatedict_vstems.ufo] 
tests/integration/test_hint.py::test_bad_privatedict_accept[tests/integration/data/dummy/bad_privatedict_vstems.otf] 
[gw7][36m [ 41%] [0m[32mPASSED[0m tests/integration/test_hint.py::test_bad_privatedict_accept[tests/integration/data/dummy/bad_privatedict_vstems.otf] 
tests/integration/test_hint.py::test_bad_privatedict_accept[tests/integration/data/dummy/bad_privatedict_hstems.otf] 
[gw7][36m [ 42%] [0m[32mPASSED[0m tests/integration/test_hint.py::test_bad_privatedict_accept[tests/integration/data/dummy/bad_privatedict_hstems.otf] 
tests/integration/test_hint.py::test_bad_privatedict_accept[tests/integration/data/dummy/bad_privatedict_hstems.ufo] 
[gw7][36m [ 42%] [0m[32mPASSED[0m tests/integration/test_hint.py::test_bad_privatedict_accept[tests/integration/data/dummy/bad_privatedict_hstems.ufo] 
tests/integration/test_hint.py::test_bad_privatedict_accept[tests/integration/data/dummy/bad_privatedict_bluevalues.ufo] 
[gw7][36m [ 43%] [0m[32mPASSED[0m tests/integration/test_hint.py::test_bad_privatedict_accept[tests/integration/data/dummy/bad_privatedict_bluevalues.ufo] 
tests/integration/test_hint.py::test_ok_privatedict_accept[tests/integration/data/dummy/ok_privatedict_hstems.otf] 
[gw4][36m [ 43%] [0m[32mPASSED[0m tests/integration/test_cli.py::test_cid_glyph_list[cid0-cid10] 
tests/integration/test_cli.py::test_exclude_glyph_list[a,b,c] 
[gw7][36m [ 44%] [0m[32mPASSED[0m tests/integration/test_hint.py::test_ok_privatedict_accept[tests/integration/data/dummy/ok_privatedict_hstems.otf] 
tests/integration/test_hint.py::test_ok_privatedict_accept[tests/integration/data/dummy/ok_privatedict_vstems.otf] 
[gw7][36m [ 45%] [0m[32mPASSED[0m tests/integration/test_hint.py::test_ok_privatedict_accept[tests/integration/data/dummy/ok_privatedict_vstems.otf] 
tests/integration/test_hint.py::test_ok_privatedict_accept[tests/integration/data/dummy/ok_privatedict_hstems.ufo] 
[gw7][36m [ 45%] [0m[32mPASSED[0m tests/integration/test_hint.py::test_ok_privatedict_accept[tests/integration/data/dummy/ok_privatedict_hstems.ufo] 
tests/integration/test_hint.py::test_ok_privatedict_accept[tests/integration/data/dummy/ok_privatedict_vstems.ufo] 
[gw7][36m [ 46%] [0m[32mPASSED[0m tests/integration/test_hint.py::test_ok_privatedict_accept[tests/integration/data/dummy/ok_privatedict_vstems.ufo] 
tests/integration/test_hint.py::test_too_long_glyph_name 
[gw7][36m [ 46%] [0m[32mPASSED[0m tests/integration/test_hint.py::test_too_long_glyph_name 
tests/integration/test_hint.py::test_hashmap_glyph_changed 
[gw7][36m [ 47%] [0m[32mPASSED[0m tests/integration/test_hint.py::test_hashmap_glyph_changed 
tests/integration/test_hint.py::test_hashmap_processed_no_autohint 
[gw7][36m [ 47%] [0m[32mPASSED[0m tests/integration/test_hint.py::test_hashmap_processed_no_autohint 
tests/integration/test_hint.py::test_hashmap_no_version 
[gw7][36m [ 48%] [0m[32mPASSED[0m tests/integration/test_hint.py::test_hashmap_no_version 
tests/integration/test_hint.py::test_hashmap_old_version 
[gw7][36m [ 48%] [0m[31mFAILED[0m tests/integration/test_hint.py::test_hashmap_old_version 

=================================== FAILURES ===================================
[31m[1m___________________________ test_hashmap_old_version ___________________________[0m
[gw7] linux -- Python 3.10.9 /nix/store/sp5x6s8n36gjlwck74xhj1i61p66vcpa-python3-3.10.9/bin/python3.10

tmpdir = local('/build/pytest-of-nixbld/pytest-0/popen-gw7/test_hashmap_old_version0')
caplog = <_pytest.logging.LogCaptureFixture object at 0x7ffff4416d70>

    def test_hashmap_old_version(tmpdir, caplog):
        caplog.set_level(logging.INFO)
        path = "%s/dummy/hashmap_old_version.ufo" % DATA_DIR
        out = str(tmpdir / basename(path)) + ".out"
        options = Options(path, out)
    
        hintFiles(options)
        msgs = [r.getMessage() for r in caplog.records]
>       assert "Updating hash map: was older version" in msgs
[1m[31mE       AssertionError: assert 'Updating hash map: was older version' in ['Hinting font tests/integration/data/dummy/hashmap_old_version.ufo. Start time: Tue Jan 31 20:34:04 2023.', 'Done with font tests/integration/data/dummy/hashmap_old_version.ufo. End time: Tue Jan 31 20:34:04 2023.'][0m

[1m[31mtests/integration/test_hint.py[0m:293: AssertionError
------------------------------ Captured log call -------------------------------
INFO     psautohint.autohint:autohint.py:823 Hinting font tests/integration/data/dummy/hashmap_old_version.ufo. Start time: Tue Jan 31 20:34:04 2023.
INFO     psautohint.autohint:autohint.py:841 Done with font tests/integration/data/dummy/hashmap_old_version.ufo. End time: Tue Jan 31 20:34:04 2023.
[36m[1m=========================== short test summary info ============================[0m
[33mSKIPPED[0m [3] tests/integration/test_cli.py:114: 'tx' is missing
[31mFAILED[0m tests/integration/test_hint.py::[1mtest_hashmap_old_version[0m - AssertionError: assert 'Updating hash map: was older version' in ['Hinting ...
[31m!!!!!!!!!!!!!!!!!!!!!!!!!! stopping after 1 failures !!!!!!!!!!!!!!!!!!!!!!!!!!![0m
!!!!!!!!!!!! xdist.dsession.Interrupted: stopping after 1 failures !!!!!!!!!!!!!
[31m=================== [31m[1m1 failed[0m, [32m84 passed[0m, [33m3 skipped[0m[31m in 14.43s[0m[31m ===================[0m
/nix/store/3yfs41f4b60jya2gk6xikx4s97zsxjr0-stdenv-linux/setup: line 1573: pop_var_context: head of shell_variables not a function context
```

### Additional context
Add any other context about the problem here.

### Notify maintainers
<!--
Please @ people who are in the `meta.maintainers` list of the offending package or module.
If in doubt, check `git blame` for whoever last touched something.
-->

@sternenseemann

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.1.8-JohnOS, NixOS, 23.05 (Stoat), 23.05.20230128.fc4d2ab`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.14.0pre20230127_ccaadc9`
 - nixpkgs: `/nix/store/x4dpvyfdvgzasbzay1zbyw4g7k627zb2-source`

```

--- a/pkgs/development/python-modules/psautohint/default.nix
+++ b/pkgs/development/python-modules/psautohint/default.nix
@@ -44,6 +44,9 @@ buildPythonPackage rec {
     "test_multi_outpath"
     "test_mmhint"
     "test_otf"
+    # flaky tests (see https://github.com/adobe-type-tools/psautohint/issues/385)
+    "test_hashmap_old_version"
+    "test_hashmap_no_version"
   ];
 
   passthru.tests = {

